### DISCUSSION TITLE
GSoC: Reaching out to mentors of Project 7 (OpenVINO adapters - PyTorch adapter)

### DISCUSSION THREAD

**Discussion by LucaTamSapienza** [openvinotoolkit/openvino#23176] [Google Summer of Code]
Hi, I'm Luca, I am currently graduating in computer engineering at Sapienza University of Rome. I would like to discuss some details about this project and, if possible, receive some feedback about my application before submitting it. 
Thanks for your help

**Comment by adrianboguszewski** [comment]
Hey @LucaTamSapienza. The mentors are @jiwaszki and @akuporos

**Reply by LucaTamSapienza** [reply]
Thank you so much for your response, @adrianboguszewski ! I have filled out the application template and am currently working on writing the abstract and organizing a timeline for completion. I have a few questions regarding the implementation. Would it be possible to discuss them and receive feedback on the application before submitting it?

**Reply by adrianboguszewski** [reply]
Yes. It's ok to discuss with the mentors now.

**Reply by jiwaszki** [reply]
@LucaTamSapienza Hi! Can you send me an email with your questions/application? Let's talk about it!

**Reply by LucaTamSapienza** [reply]
Sure, thanks for your time!
